Components/model.css

```diff
@@ -0,0 +1,24 @@
+.model-container {
+  position: relative;
+}
+
+.model-pop-up {
+    border : solid  var(--blue) 1px;
+    margin : 1rem;
+    z-index : 9;
+    padding : 0.5rem 1rem;
+    display : flex;
+    flex-direction : column;
+    max-width : 40rem;
+  }
```

Comment:
You can define flex related classes separately can use where you want.

Reply:
thanks